sdk/python/feast/permissions/client/utils.py

```diff
@@ -7,11 +7,11 @@
 
 def create_auth_header(
```

Comment:
sorry for not catching this before, but since this is a grpc-specific function (REST client has it's own code and does not need a list of tuples), can we rename it?

Reply:
Its GRPC and ARROW specific, how do you suggest to name it?

Reply:
Now that I look at it I think we can change Rest Client to use it also